.github/workflows/test-helpers/make-issue

```diff
@@ -54,9 +72,23 @@ function backoff {
   exit 1
 }
 
-echo "Downloading and extracting the validation workflow run log for issue #$1."
-backoff download-and-extract "$1"
+issue_number=$(
+  basename $(
+    gh issue create --title "Greetings, program!" --body "$1"
+  )
+)
+echo "Downloading and extracting the validation workflow run log for issue #${issue_number}."
+backoff download-and-extract-log "$issue_number"
```

Comment:
I guess we no longer need this step?

Reply:
Definitely still needed! This is how we guarantee that the workflows associated with the issue creation are complete, and therefore the final state of the issue is ready for testing.